HW13/task1.py

```diff
@@ -0,0 +1,88 @@
+import json
+
+
+def stats():
+    print(len(telephone_book))
+
+
+def adds():
+    name = input("Please enter name and last name: ")
+    telephone = input("Please enter telephone number: ")
+    # checking whether the entered data is correct and whether this name is not in the phone book
+    if len(name) < 4 or len(telephone) < 9:
+        print("Incorrect name or phone number, please check and try again.")
+    elif name in telephone_book:
+        print("This contact is already in the phone book.")
+    else:
+        telephone_book[name] = telephone
+        # update the json file
+        json_telephone_book = json.dumps(telephone_book)
+        with open("json_telephone_book.json", "w") as file:
+            file.write(json_telephone_book)
```

Comment:
nit: better to move writing to file in own function to avoid duplication of code in many places

Reply:
ok, i will change it